platform-hub-web/src/app/shared/hub-api/hub-api.service.js

```diff
@@ -682,6 +683,28 @@ export const hubApiService = function ($rootScope, $http, $q, logger, events, ap
     return msg;
   }
 
+  function allocateKubernetesGroup(groupId, projectId, serviceId) {
```

Comment:
Don't you need a default `null` for serviceId?

Reply:
Not necessarily, as it will be `undefined` if not passed in (such dynamic JavaScript ;-).